runtime-linux.md

```diff
@@ -1,11 +1,5 @@
 # Linux Runtime
 
-## File descriptors
-
-By default, only the `stdin`, `stdout` and `stderr` file descriptors are kept open for the application by the runtime.
-The runtime MAY pass additional file descriptors to the application to support features such as [socket activation](http://0pointer.de/blog/projects/socket-activated-containers.html).
-Some of the file descriptors MAY be redirected to `/dev/null` even though they are open.
```

Comment:
> How do you know there aren't meaningful standard streams for web users?

Your example is the point I was making. You **can** have IO with a container through a different interface, but the `stdio` will **not** be hooked up to a **terminal**. Quoting your previous response:

> but I think it's redundant **when we have the command line API** covering both the presence of these file descriptors when executing the container process and **telling you how to set them up** [emphasis added]

The command-line API does **not** tell web-interfaces how to set up stdio, because there isn't a "current terminal" with a web interface (well there is, but it's the stdio for the server rather than the stdio interface for the website). So we need this part to mention that you are always guaranteed to have `/proc/self/{0,1,2}` open as stdio -- but we don't mention what exactly they're open to (if you want concrete information use the console API).

> As folks add additional APIs, I'd expect them to also address how the container process's standard streams are wired up in that API. Telling image authors

How does removing this paragraph tell image creators anything more helpful? You've **removed** the information. As for the rest of your comment, I mostly agree with your points and I think your final wording is good. The `/dev/null` point was an example of an edge case where the old wording didn't work, but this wording LGTM:

> > All OCI-specified runtime APIs MUST allow runtime-callers to specify the file descriptors which will be inherited by the container process, with the POSIX exec caveat that if file descriptor 0, 1, or 2 would otherwise be closed, the container process may have unspecified files opened for those file descriptors.